app/models/subscription.rb

```diff
@@ -1,10 +1,7 @@
 class Subscription < ApplicationRecord
   include Auditor::Model
 
-  FREQUENCY_OPTIONS = {
-    daily: 0,
-    weekly: 1,
-  }.freeze
+  FREQUENCY_OPTIONS = { daily: 0, weekly: 1 }.freeze
```

Comment:
You do not need this. You can just have
```
enum frequency: { daily: 0, weekly: 1 }
```

and then you access the options with:
```
Subscription. frequencies
```